i18n/extract.py

```diff
@@ -316,9 +316,9 @@ def fix_metadata(pofile):
         'Last-Translator': '',
         'Language-Team': 'openedx-translation <openedx-translation@googlegroups.com>',
         'Plural-Forms': 'nplurals=2; plural=(n != 1);',
+        'POT-Creation-Date': '2023-06-13 08:00+0000',
+        'PO-Revision-Date': '2023-06-13 09:00:00.000000',
     }
-    pofile.metadata.pop('POT-Creation-Date', None)
-    pofile.metadata.pop('PO-Revision-Date', None)
```

Comment:
if you restore this line @OmarIthawi , the integration test will fail. `pofile.metadata.pop('PO-Revision-Date', None)`

Restoring the creation date will not fail the test because it's not being checked by `validate` anyway

Reply:
> Restoring the creation date will not fail the test because it's not being checked by validate anyway

Please feel free to restore only what's required by `validate`. I wouldn't rely on my intuition for this obscure validation part.

Reply:
I think we're good, let's keep them both